infra/growth/src/apollo/resolvers.js

```diff
@@ -169,10 +169,20 @@ const resolvers = {
           )
         }
 
-        return { authToken }
+        return {
+          authToken,
+          isBanned: false
+        }
       } catch (e) {
-        logger.warn('User authentication failed: ', e.message, e.stack)
-        throw new AuthenticationError('Growth authentication failure')
+        if (e.message === 'This user is banned') {
```

Comment:
I'm slightly worried that in the future someone may change the error message emitted and it will break.
To make this less breakage prone, what do you think of defining a new exception type and relying on e.name ?

Code sample:
```
class BannedUserError extends Error {
  constructor(msg) {
    super(msg)
    this.name = this.constructor.name
   }
 }

try {
  throw new BannedUserError(`User ${ethAddress} is banned`)
} catch (e) {
  if (e.name === 'BannedUserError') {
    console.log('Banned user error !', e.message)
  }
}
```

Reply:
Yeah that is a good point - your way is much more resilient... making the correction